frontend/pages/journey/[id].vue

```diff
@@ -0,0 +1,441 @@
+                        "
+                      />
+                    </div>
+                    <div class="w-full h-[1/3] col-span-2 relative">
+                      <div
+                        class="absolute ml-10 rounded-full border-input-placeholder text-input-placeholder w-16 h-16 self-center border-dashed border-2 right-2 bottom-2 flex text-center justify-center text-xs pl-1.5 pr-1.5 pt-1 dark:border-white dark:text-white"
+                      >
+                        <T keyName="journey.turn" />
+                      </div>
+                      <SvgStripes class="h-fit" />
+                    </div>
+                  </div>
+                </div>
+                <div
+                  class="h-[90%] w-0 rounded-b-r-3xl border-border-gray border-r-2 border-dashed"
+                ></div>
+              </div>
+            </div>
+            <div
+              class="text-text absolute inset-0 h-full w-full rounded-xl bg-white text-center [transform:rotateX(180deg)] [backface-visibility:hidden] dark:bg-background-dark"
+            >
+              <div
+                class="bg-border border-x-2 border-t-2 border-border-darker rounded-t-2xl flex items-center relative dark:border-border-blue-dark dark:bg-ticket-top-dark-bg"
+              >
+                <div
+                  class="absolute ml-10 rounded-full w-7 h-7 bg-border-gray inline-block self-center"
+                ></div>
+                <p class="ml-20 text-white font-bold text-xl">JourneyPlanner</p>
+                <div class="w-full flex justify-end h-full items-center">
+                  <SvgAirplaneIcon class="w-9 mr-5 pb-1" />
+                </div>
+              </div>
+              <div class="flex h-5/6">
+                <div
+                  class="h-full w-full rounded-b-2xl bg-background border-border-gray border-x-2 border-b-2 flex justify-center dark:bg-border-dark dark:border-form-input-dark"
+                >
+                  <div class="h-full w-full flex flex-col items-end relative">
+                    <img
+                      class="absolute right-[50%] top-[25%] z-20 translate-x-[50%] -translate-y-[25%] w-2/5"
+                      :src="qrcode"
+                      alt="QR Code"
+                    />
+                    <div
+                      class="absolute ml-10 rounded-full border-input-placeholder text-input-placeholder w-16 h-16 self-center border-dashed border-2 right-2 bottom-2 flex text-center justify-center text-xs pl-1.5 pr-1.5 pt-1 z-40 dark:border-white dark:text-white"
+                    >
+                      <T keyName="journey.turn" />
+                    </div>
+                    <SvgStripes class="z-0 lg:w-1/2 md:w-2/3" />
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div
+        class="ml-[10%] lg:w-1/3 md:w-[50%] md:visible invisible w-0 max-md:h-0"
```

Comment:
ca hier glaube

ganze karte/box ding hat zu viel Höhe und wirkt dadurch nicht so wie das "Ticket" in Figma

Stand jetzt (mit bereits paar Sachen bearbeitet von den anderen Comments z.B. Header davon schon kleiner, trotzdem noch zu hoch):
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/6a996066-aadc-46af-a9c5-51430e417b1e)

Figma:
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/e48efb6f-2644-44c9-bca1-06aae1b65a31)


Sollte Höhe anpassen GAR nicht gehen, dann die rechte Box mit QRCode etwas breiter, damit sie auch wirklich zu einem Quadrat wird (so wie in Figma) (also sollte generell ein Quadrat sein, auch wenn dus schaffst Höhe zu bearbeiten)